components/region_cache_memory_engine/src/background.rs

```diff
@@ -420,6 +424,38 @@ impl Runnable for BackgroundRunner {
     }
 }
 
+#[derive(Default)]
+struct FilterMetrics {
+    total: usize,
+    versions: usize,
+    delete_versions: usize,
+    filtered: usize,
+    unique_key: usize,
+    mvcc_rollback_and_locks: usize,
+}
+
+impl FilterMetrics {
+    fn merge(&mut self, other: &FilterMetrics) {
+        self.total += other.total;
+        self.versions += other.versions;
+        self.delete_versions += other.delete_versions;
+        self.filtered += other.filtered;
+        self.unique_key += other.unique_key;
+        self.mvcc_rollback_and_locks += other.mvcc_rollback_and_locks;
+    }
+
+    fn flush(&self) {
+        GC_FILTERED_STATIC.total.inc_by(self.total as u64);
```

Comment:
How about using `GC_FILTERED_STATIC.local()`, then you don't need `FilterMetrics` and flush manually.

Reply:
Gc is performed only in one thread. Why I merge FilterMetrics here is to make the metrics reflects the whole range of the in-memory engine.

Reply:
You still can use `GC_FILTERED_STATIC.local()` which reflects the whole range of the in-memory engine.

Reply:
If it's for performance, I thinks it's fine. Only three minutes(default) will flush once.

Reply:
I think it's better to use GC_FILTERED_STATIC.local() regardless. 
@SpadeA-Tang

Reply:
MIO, local is used when multiple threads are updating the metrics so that the metrics are batched in local and the conention is reduced. But here these's only one thread updating this metrics for every three minutes which means we do not need to reduce contention as well as batch. In addition, using local may involve extra overhead.

Reply:
I think `GC_FILTERED_STATIC.local()` is essentially the same as `FilterMetrics`. You just don't need to reimplement `FilterMetrics`.

Reply:
Ok, I got what you mean now.